src/file_reader.cpp

```diff
@@ -84,7 +84,7 @@ char MultiPartFileReader::read(offset_t offset) const {
     s << " - physical local offset is " << physical_local_offset.v << "\n";
```

Comment:
Should I change '\n's used with Formatter to std::endl as well? Or do these server a specific purpose?